gaia-media-controls.js

```diff
@@ -0,0 +1,291 @@
+   *    elapsed time, and play head.
+   */
+  .play-head {
+    top: calc(50% - (1.15rem + 0.4rem));
+    position: relative;
+    width: 2.3rem;
+    height: 2.3rem;
+
+    /* For LTR langauges, position the playhead 1.15 rems to the left
+     * so that the center of the playhead aligns with the beginning of
+     * the time slider.
+     */
+    margin-left: -1.15rem;
+
+    /* For RTL langauges, position the playhead 1.15 rems to the right
+     * so that the center of the playhead aligns with the end of
+     * the time slider.
+     */
+    margin-right: -1.15rem;
+
+    border: none;
+    background: none;
+    pointer-events: none;
+    z-index: 30; /* 2 */
+  }
+
+  /*
+   * Define the 'normal' play-head graphic. Using the 'after' pseudo-element
+   * here specifies that the 'normal' (smaller, white) play-head will
+   * appear on top of the larger, blue 'active' play-head (specified using
+   * the 'before' pseudo-element).
+   */
+  .play-head:after {
+    content: "";
+    position: absolute;
+    top: 0;
+    left: calc(50% - 1.15rem);
+    width: 2.3rem;
+    height: 2.3rem;
+    border-radius: 50%;
+    background-color: #fff;
+  }
+
+  /* Define the 'active' play-head graphic (blue, larger than the 'normal'
+   * play-head). Using the 'before' pseudo-element specifies that the 'active'
+   * play-head will appear under the 'normal' play-head.
+   */
+  .play-head.active:before {
+    content: "";
+    position: absolute;
+    top: calc(50% - 3.05rem);
+    left: calc(50% - 3.05rem);
+    width: 6.1rem;
+    height: 6.1rem;
+    border-radius: 50%;
+    background-color: #00CAF2;
+  }
+
+  /* video control bar -- rewind, pause/play, forward
+   *
+   * 1. The buttons should always display left-to-right.
+   */
+  .video-control-bar {
+    display: flex;
+    flex-direction: row;
+    flex-basis: 4.8rem;
+    border-top: 0.1rem solid rgba(255,255,255, 0.1);
+    background-color: rgba(0,0,0, 0.95);
+    overflow: hidden;
+    direction: ltr; /* 1 */
+    /*z-index: 10*/;
+  }
+
+  .seek-backward,
+  .seek-forward,
+  .play {
+    /* All three elements grow and shrink together by the same proportion */
+    flex-grow: 1;
+    padding: 0;
+  }
+
+  .play.paused:before {
+    content: 'play';
+    padding-left: 4px;
+  }
+
+  .player-controls-button {
+    color: #FFFFFF;
+    border: none;
+    border-radius: 0;
+    background: transparent;
+  }
+
+  .player-controls-button:hover {
+    background: transparent;
+  }
+
+  .player-controls-button:active {
+    background: #00caf2;
+  }
+
+  .player-controls-button:disabled {
+    opacity: 0.3;
+  }
+
+  .player-controls-button:disabled:active {
+    background: transparent;
+  }
+
+  </style>
+
+  <div class="media-controls-container">
+    <div class="time-slider-bar">
+      <span class="elapsed-text"></span>
+      <div class="slider-wrapper">
+        <div class="elapsed-time progress"></div>
+        <div class="time-background progress"></div>
```

Comment:
I suspect you can just move time-background to be before elapsed-time and that should get the z-index stuff sorted out.